Dockerfile

```diff
@@ -1,30 +1,78 @@
 FROM tailor/java-oracle:latest
 MAINTAINER Nadav Shatz <nadav@tailorbrands.com>
 
-RUN apt-get update
-RUN apt-get install -y wget
-RUN apt-get install -y libc6-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN add-apt-repository -y ppa:webupd8team/java
 
-# jruby
-ENV JRUBY_VERSION 9.1.8.0
-ENV JRUBY_SHA256 20ac501c99a7cb3cf53ded64ac1b8bb6e0b0f6ba34a41b8bacc9715cd4bb2601
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
```

Comment:
should be changed from `trust` to `xenial` no?
alternatively you can change `trusty` to `$(lsb_release -cs)` and it'll be future proof.

Reply:
👍